Dockerfile

```diff
@@ -4,15 +4,20 @@ USER root
 
 WORKDIR /opt/app-root/src/
 
-# Install socat tools required by acme.sh and download acme.sh (but don't install)
+ARG ACME_RELEASE_TAG=3.0.5
+ARG DRIVER_RELEASE_TAG=dev
```

Comment:
Even w/ dev it might be useful to set it to the current git hash?  Or at least date/time.

Reply:
Yes, I wanted to get current git commit SHA but would need to run `$(shell ....)` in make file for that **BUT** that's not supported in all versions of make. 
Hence the approach of default value in `Dockerfile` is `dev` (in case someone builds it directly without make) and when invoked from `make`, it will get value of "$IMAGE_TAG". 

With all that, in development we can call like shown below and get the latest SHA in there.
```sh
IMAGE_TAG=$(git rev-parse --short HEAD) make assemble
```